src/desktop/apps/articles/components/__tests__/AuthWrapper.jest.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react"
+import { mount } from "enzyme"
+import { AuthWrapper } from "../AuthWrapper"
+import { data as sd } from "sharify"
+
+jest.mock("sharify")
+jest.mock("querystring", () => ({
+  parse: jest.fn().mockReturnValue({}),
+}))
+jest.mock("desktop/components/cookies/index.coffee", () => ({
+  get: jest.fn(),
+  set: jest.fn(),
+}))
+jest.mock("desktop/lib/mediator.coffee", () => ({
+  trigger: jest.fn(),
+  on: jest.fn(),
+}))
+const mediatorTrigger = require("desktop/lib/mediator.coffee")
+  .trigger as jest.Mock
+const mediatorOn = require("desktop/lib/mediator.coffee").on as jest.Mock
+const qsMock = require("querystring").parse as jest.Mock
+const CookiesGetMock = require("desktop/components/cookies/index.coffee")
+  .get as jest.Mock
+const CookiesSetMock = require("desktop/components/cookies/index.coffee")
+  .set as jest.Mock
+
+jest.useFakeTimers()
+
+describe("AuthWrapper", () => {
+  beforeEach(() => {
+    delete sd.IS_MOBILE
+    delete sd.CURRENT_USER
```

Comment:
Will this cause cascading side-effects in other tests? Maybe restore `sd` values in an `after` block?

Reply:
(Ah, I see below)